common/source/docs/common-autopilots.rst

```diff
@@ -1,6 +1,57 @@
 .. _common-autopilots:
 
-==========================
+=====================
+Choosing an Autopilot
+=====================
+
+ArduPilot runs on many different autopilot boards, the most
+important of which are linked from the topic :ref:`AutoPilot Hardware Options <common-autopilots>`.
+
+Selecting the right board depends on the physical restraints of the
+vehicle, features desired, and the applications that you want to run.
+Factors to consider are:
+
+#. Sensor Redundancy: ArduPilot supports redundant IMUS, GPS, etc. Many controllers have multiple IMUs integrated on board.
+#. Number of Servo/Motor Outputs
+#. Number of UARTs: Telemetry radios, GPS's, Companion Computers, etc can be attached via these ports
+#. External Buses: I2C, CAN, etc. allow many types of devices, such as airspeed sensors, LED controllers, etc. to be attached to the autopilot.
+#. Number of Analog I/O: Some controllers have analog I/O available for such features as inputting receiver signal strength (RSSI) or battery voltage/current or other analog sensors.
+#. Integrated Features: Such as on-board OSD (On Screen Display), integrated battery monitoring sensors
+#. Size: Many vehicles have limited space for the autopilot.
+#. Expense: Controller prices range from ~$25 to much more, depending on feature set.
+
+ Broadly speaking:
```

Comment:
I  think we're stepping into a mine field with the "Broadly speaking" section.  I think the best we could do is tell people to ask in the forums.

Reply:
historical, but can change